### ISSUE TITLE
[Bug Report] [Component] [popconfirm] Pop confirm aria-describedby target missing

### ISSUE THREAD

**Issue by plehnen** [element-plus/element-plus#8155]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.4`
- Browser / OS: `Chrome (latest)`
- Build Tool: `Vue CLI`

### Related Component

- `el-popconfirm`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/popconfirm.html)

### Steps to reproduce

Look at the DOM of the Pop-Confirm component.

### What is Expected?

The popover should have the defined "id" property.

### What is actually happening?

The button has the property `aria-describedby="el-id-8705-51"`, but in the DOM there is no element with this id.
This is not only invalid html, it also breaks all our tests, because we are searching for this id-element and look for the buttons inside.

### Additional comments

_(empty)_

**Comment by plehnen** [comment]
Also applies for ElPopover component :(

**Comment by CisageX** [comment]
Yes, development env 's console warning

[Vue warn]: Extraneous non-props attributes (id, role, tabindex, class, aria-controls, aria-describedby, aria-expanded, aria-haspopup) were passed to component but could not be automatically inherited because component renders fragment or text root nodes.

**Comment by CisageX** [comment]
I just tried, version 2.2.2 is good, this problem occurs in version 2.2.3 and later

**Comment by CisageX** [comment]
this bug still exists

**Comment by plehnen** [comment]
Yes. It is not possible to update from 2.2.2 until this is fixed :(

But apparently no developer is taking care of it yet. Is there a way to push this with more priority? or to gain the attention of the element-plus developers? maybe creating a new issue and linking this? I'd love to get at least some feedback if this is going to be fixed soon...

**Comment by plehnen** [comment]
So I created a PR for this: https://github.com/element-plus/element-plus/pull/9632
No reviewer since 5 days. Hope someone responsible can have a look. It's just two lines of code that changed.

**Comment by Mercer-L** [comment]
It seems like this bug has not been fixed yet.

**Comment by plehnen** [comment]
@Mercer-L I just checked the Docs page and could successfully inspect the ID is correctly addressed inside aria-describedby and the associated div does exist in the DOM. so to me it still looks fixed.
